src/main/java/org/mozilla/jss/netscape/security/x509/RevocationReasonAdapter.java

```diff
@@ -24,8 +24,10 @@
 /**
  * The RevocationReasonAdapter class provides custom marshaling for RevocationReason.
  *
+ * @deprecated Use the revocation reason code or label instead.
  * @author Endi S. Dewata
  */
+@Deprecated
```

Comment:
Seeing as we know we are removing this we should provide this information to future archaeologists:
`@Deprecated(since="4.9.1", forRemoval=true)`

Reply:
Nice! I wasn't aware of this feature. Patch updated.